src/main/java/org/prebid/server/auction/requestfactory/AuctionRequestFactory.java

```diff
@@ -232,6 +232,12 @@ private Future<BidRequest> updateBidRequest(AuctionStoredResult auctionStoredRes
     }
 
     private static MetricName requestTypeMetric(BidRequest bidRequest) {
-        return bidRequest.getApp() != null ? MetricName.openrtb2app : MetricName.openrtb2web;
+        if (bidRequest.getApp() != null) {
+            return MetricName.openrtb2app;
+        } else if (bidRequest.getDooh() != null) {
```

Comment:
Super small but we expect to have 99.99% requests of site and app types
So please make if(site) before if(dooh)
Also I think it would be great if we will throw some exception by default, and have explicit checks for each mediaType before this default statement

Reply:
imho an exception here gonna be duplicate the request validator logic